### PR TITLE
Features

### PR DESCRIPTION
**Pull request by ignatius22**
## Home
     - navbar
     - weather details

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const getWeatherData = (() => {
+  const getWeather = async (city, unit = 'F') => {
+    const API_KEY = '3bcff5eb7f5c2ce7a067f51ae1ab8863';
+    const URL = 'http://api.openweathermap.org/data/2.5/weather?q';
+    const getData = await fetch(`${URL}=${city}&units=${unit}&APPID=${API_KEY}`,
+      { mode: 'cors' });
+    const weatherData = await getData.json();
+    return weatherData;
+  };
```
Comment (Dandush03) [inline src/data.js]: ### - [ ] It has error handling for async/await

File: README.md [hunk 2]
```diff
@@ -1,11 +1,63 @@
...
-Screenshots
\ No newline at end of file
+# weather app
+
+This is a weather forecast site built using a weather API. Requirements can be seen from The [Odin Curriculum](https://www.theodinproject.com/courses/javascript/lessons/weather-app).
+
+## Live demo
+
+[live demo](https://adoring-swirles-ca3ccf.netlify.app/)
+
```
Comment (Dandush03) [inline README.md]: ### - [ ] Deployment
Wrong link

**Review by Dandush03 — CHANGES_REQUESTED** [review]
# Status: :red_circle: Require Changes :red_circle: 
### Great work so far :muscle:, Check out my comments below before submiting for a next review.
# :computer:  Happy Coding :nerd_face:

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @ignatius22, congratulations on fixing all the comments outlined by the previous TSE :fireworks: 
The project now meets all the minimum requirements :smile: 

Go ahead and merge into development :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**